src/GafferUI/View.cpp

```diff
@@ -268,16 +268,36 @@ void View::toolPlugSet( Gaffer::Plug *plug ) const
 		return;
 	}
 
-	if( !tool->activePlug()->getValue() || !exclusive( tool ) )
+	if( !exclusive( tool ) )
 	{
 		return;
 	}
 
+	if( tool->activePlug()->getValue() )
+	{
+		for( auto &t : Tool::Range( *tools() ) )
+		{
+			if( t != tool && exclusive( t.get() ) )
+			{
+				t->activePlug()->setValue( false );
+			}
+		}
+	}
+
+	bool exclusivePlugActive = false;
 	for( auto &t : Tool::Range( *tools() ) )
 	{
-		if( t != tool && exclusive( t.get() ) )
+		exclusivePlugActive |= ( exclusive( t.get() ) && t->activePlug()->getValue() );
+	}
+	if( !exclusivePlugActive )
+	{
+		for( auto &t : Tool::Range( *tools() ) )
 		{
-			t->activePlug()->setValue( false );
+			auto order = Metadata::value<IntData>( t.get(), "order" );
+			if(  order && order->readable() == 0 )
+			{
+				t->activePlug()->setValue( true );
+			}
```

Comment:
`break`?

Reply:
Given a `break` in https://github.com/GafferHQ/gaffer/pull/5518/commits/34a7926dd0b395ac854135772f5ec5a783c92493